nordic-nrf51822-16k-armcc/scripts/memory_info.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env python
```

Comment:
why do you run `python` within the context of `/usr/bin/env`? Is this cross-platform?